Vertexvis/vertex-web-sdk
# [PLAT-3139][PLAT-2057] Using vertex instead of vertexvis for resources

## Summary

The vertexvis urn namespace is unneeded. We want it to just be vertex. 

The `vertexvis` namespace is deprecated, but we will still support it. 

Additionally changed the orthographic thresholds to be a little more or less snappier for the zoom/pan gestures.

## Test Plan

- Verify both `vertexvis` and `vertex` are allowed for namespaces (ensuring backwards compatibility)
- Verify that the `vertexvis` namespace logs a warning in the developer console. 


## Release Notes


## Possible Regressions


## Dependencies
